core/chains/evm/gas/models.go

```diff
@@ -271,12 +271,25 @@ func (e *evmFeeEstimator) GetFee(ctx context.Context, calldata []byte, feeLimit
 		fee.DynamicTipCap = dynamicFee.TipCap
 		return
 	}
-
 	// get legacy fee
 	fee.Legacy, chainSpecificFeeLimit, err = e.EvmEstimator.GetLegacyGas(ctx, calldata, feeLimit, maxFeePrice, opts...)
 	if err != nil {
 		return
 	}
+
+	if e.geCfg.CostMax() != nil {
```

Comment:
So this pattern assumes that for chains where we don't set CostMax, will omit it from their toml config.

Another pattern I've seen is to set CostMax = 0 in the fallback.toml.
0 means no limit.
Any chain that wants a limit, can set it in its own config.

I think we generally use the 2nd pattern in other places.

@jmank88 any thoughts?

Reply:
Having `0` mean unlimited sounds nicer, but for `PriceMax` we use the literal max value: PriceMax = '115792089237316195423570985008687907853269984665.640564039457584007913129639935 tether'

I don't recall why exactly that is :thinking:

Reply:
Added the 0= no limit pattern